Makefile

```diff
@@ -10,6 +10,14 @@ libgit2:
 	-git submodule update --init --recursive
 	cd git2go; make install-static
 
+# build and install libgit2 for macOS m1
```

Comment:
nice, I didn't know this would work for libgit2... @dwertent can confirm on his M1

Reply:
Yeah, that do need further confirmation. So far I can only see that this resolves the build error here: https://github.com/HollowMan6/kubescape/actions/runs/4416191690/jobs/7740157266